lib/audit/catalog_to_moab.rb

```diff
@@ -8,8 +8,10 @@ class CatalogToMoab
   def self.check_version_on_dir(last_checked_b4_date, storage_dir)
     # TODO: ensure last_checked_version_b4_date is in the right format for query - see #485
     pcs = PreservedCopy
-          .where('last_version_audit < ? OR last_version_audit IS NULL', last_checked_b4_date)
-          .order('last_version_audit IS NOT NULL, last_version_audit ASC')
+          .joins(:endpoint)
+          .where(endpoints: { storage_location: storage_dir })
+          .where('last_version_audit IS NULL or last_version_audit < ?', last_checked_b4_date)
```

Comment:
ooh, actually, i'd be slightly in favor of putting the `.order` back in, though i don't feel strongly about it (and would be fine with ticketing it if you found that suggestion agreeable but didn't want to do it in this PR).